### ISSUE TITLE
Add function to save fingreprint templates to file

### ISSUE THREAD

**Issue by adrobinoga** [adrobinoga/pyzatt#8]
- Add functions to save and load fingerprints from a given file
- Add example test for this functionality

**Comment by vmisiko** [comment]
Yes please i have been longing for such an issue to be addressed, a function to generate fingureprint template that can be saved in a file or database. And a function that can retrieve it back to verify the fingureprint from the one one on live capture

**Comment by adrobinoga** [comment]
Sorry for delays, I'm implementing simulator to perform this tests, simulator is almost ready, maybe tomorrow I will have this feature, but you must be aware that even when you store a fingerprint template on your pc, you must upload that fingerprint back to the device and assign all the proper permissions, i.e. device doesn't returns a fingerprint template when someone attempts to register (at least on TFT devices).
